src/sinks/util/buffer/json.rs

```diff
@@ -0,0 +1,103 @@
+use crate::sinks::util::Batch;
+use serde_json::value::RawValue;
+
+type BoxedRawValue = Box<RawValue>;
```

Comment:
```
$ irb
irb(main):001:0> "BoxedRawValue".length
=> 13
irb(main):002:0> "Box<RawValue>".length
=> 13
```
:smile:

Reply:
I am considering just making the type alias public since that is easier to reach for than `serde_json::value::RawValue` then boxing it.

Reply:
Are you planning to use it in other places? Do you just not want to import it in those places? I remain confused.

Reply:
Yes, I have a plan to use it in `build_request(events: Vec<BoxedRawValue>)`.

Reply:
Could we not use `finish` to make the output such that users don't need to worry about `RawValue`?

Reply:
No it needs to be boxed since the type is DST. For encoding we need a boxed version while decoding can use the dynamically sized type behind a reference. https://docs.rs/serde_json/1.0.45/serde_json/value/struct.RawValue.html#method.from_string returns a box as well which we use.

We could newtype it but I don't think that is worth it over this.